src/utils/storageFacade.utils.ts

```diff
@@ -90,41 +175,65 @@ export async function writePackage(pkg: Package, sessionId: number) {
   return status;
 }
 
-/**
- * Read all vulnerabilities in one session
- * @param sessionId associated with one user
- * @returns list of vulnerabilities, empty list if none matching present in DB
- */
-export async function readVulnsBySession(sessionId: number) {
-  let { data, error } = await supabase
-    .from('vulnerabilities')
-    .select(
-      '*,junction!inner(packageid,packages!inner(sessionid, package_ref))'
-    )
-    .eq('junction.packages.sessionid', sessionId);
+export async function readPackagesDashboard(
+  sessionId: number,
+  sortParam: PackageViewParam,
+  riskFilters: string[],
+  page: number
+) {
+  let sortCol = mapPkgParamToColumn(sortParam);
+  let pageLowerLimit = (page - 1) * PAGE_SIZE;
+  let pageUpperLimit = page * PAGE_SIZE - 1;
+  let filterString = '';
+  for (let s of riskFilters) {
```

Comment:
add a descriptive line here - are you setting the the filter string? If so multiplication by 10 is the numerical form?